Resources/public/js/views/ez-actionbarview.js

```diff
@@ -77,7 +77,7 @@ YUI.add('ez-actionbarview', function (Y) {
                         actionList.push(
                             new Y.eZ.ButtonActionView({
                                 actionId: 'sendToTrash',
-                                disabled: false,
+                                disabled: this.get('location').get('depth') === 1,
```

Comment:
could be... but should it be here or maybe in the `location`model itself? 
nitpick: then `_isRootLocation` as we don't have nodes anymore? ;)